im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
```

Comment:
Since this is pretty unusual in parts -- is this style documented somewhere? Do you have direct contact to Ulrich Gorki or is the last name just a coincidence? We feel pretty strongly about having documentation for our styles available should issues come up. Presumably the journal sends something to authors? Could they put that online?